README.md

```diff
@@ -18,3 +18,9 @@ The proxy and resolver are configured using the file specified by the `-c` flag,
 ```bash
 $ cargo run -- example.com AAAA
 ```
+
+Instead of using `HTTPS` DNS records to retrieve configs, it can be configured to use a well known endpoint to retrieve the configs via `GET` requests. This can be done by setting the `-w` flag:
```

Comment:
Out of curiosity, why `-w`?

Reply:
Ha, "**well**-known", wanted a short flag but nothing else came to mind, you can also use the long version `use_well_known`.